dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -0,0 +1,142 @@
+--[[
+<?xml version='1.0' encoding='utf8'?>
+<mission name="Dvaered Colonel Escort">
+ <unique />
+ <priority>4</priority>
+ <chance>86</chance>
+ <location>Bar</location>
+ <faction>Dvaered</faction>
+</mission>
+--]]
+--[[
+
+   Mission: Escort a Dvaered colonel
+   Description: Small mission where you escort a Dvaered Arsenal.
+                This is a one-off that's not part of any major
+                storyline; the in-game purpose will remain a mystery.
+
+--]]
+
+local escort = require "escort"
+local fmt = require "format"
+local neu = require "neutral"
+local vn = require "vn"
+local vni = require "vnimage"
+
+local reward = 0
+
+local source_system = nil
+
+local dest_planet = spob.getS("Adham")
+
+local ffriendly, fhostile -- codespell:ignore ffriendly
+
+local npc_name = _("Dvaered Colonel")
+local npc_portrait = nil
+local npc_image = nil
+function create()
+   mem.npc_image, mem.npc_portrait = vni.dvaeredMilitary()
+   misn.setNPC(npc_name, npc_portrait, _("A Dvaered, very professional-looking, is sitting with an excellant posture at the bar.") )
+end
+
+function accept()
+   local accepted = false
+   vn.clear()
+   vn.scene()
+   local m = vn.newCharacter( npc_name {image=npc_image} )
+   vn.transition()
+   m(fmt.f([[As you approach, the Dvaered soldier stands. "Hello, captain {playername}!" they say. "Nice to see you around here. How's it going?"]]))
```

Comment:
It's one of the mysteries. I can remove it, but the idea is this Dvaered has spies working for them, and has learned information from the flight specifications or something like that.

Reply:
Mmm, if you want something like that, I would probably condition it on their reputation with Dvaered or something. Otherwise, if the player just starts out and gets to Dvaered space, it will be a bit weird. Alternative, you can make it a bit more "spy-y". Maybe something like `"Why, if it is not Captain {playername}..."`. And let the player ask "How did you know my name!?" and then you can add some sly response.